javascript/src/reducers/layers.js

```diff
@@ -14,21 +13,11 @@ const defaultLayer = {
     displayed: false
 }
 
-export default (state = [], action) => {
+const defaultState = config.seedzones.sort((a,b) => a.name.localeCompare(b.name))
```

Comment:
This is causing an exception for me:

```
TypeError: _config2.default.seedzones is undefined
```

And indeed there is no `seedzones` in `config`. Something didn't get committed/pushed?

Reply:
My plan here is to adjust the creat_vector_tiles command so it writes an export statement directly to the config file (something like export const seedzones....). Then change import at top of layers.js to something like import { seedzones } from ‘seedsource/config’; if !seedzones.length then seedzones = []. Can I go ahead with this plan or is it poor form to write directly to a config file like that?

Reply:
Yeah, I don't think we want a manage command modifying versioned code. Instead, how about writing to a `.json` file and importing that where you need it? Just make sure it gets ignored from git and make sure the code here handles a missing file gracefully (i.e., should be treated as no seedzone layers).